torchgeo/datasets/utils.py

```diff
@@ -91,6 +92,18 @@ def __init__(self, dataset: Dataset[object]) -> None:
         super().__init__(msg)
 
 
+class RGBBandsMissingError(ValueError):
+    """Raised when a dataset is missing RGB bands for plotting.
+
+    .. versionadded:: 0.5.2
+    """
+
+    def __init__(self) -> None:
+        """Instantiate a RGBBandsMissingError instance."""
+        msg = "Dataset doesn't contain some of the RGB bands"
```

Comment:
Wonder if it's possible to make this message more specific, like print the bands the user requested and which bands are RGB.

Reply:
Also, can we use "does not" instead of "doesn't"?

Reply:
I may punt on making it based on the specific bands for a future PR.